### DISCUSSION TITLE
Hemera steps modification not working in Jyers firmware

### DISCUSSION THREAD

**Discussion by hellraiserz32** [Jyers/Marlin#975] [Q&A] [answered]
I have upgraded to the hemera direct extruder.
I tried the firmware mod on stock marlin with success but when I try do do the same to Jyers firmware the extruder mod does not work.

This is the code I changed as recommended by e3d 
#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 93 }
change it to:

#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 409 }

When I change the 93 to 409 the extruder is still under extruding .. if I use stock marlin  firmware with the modification the extruder works perfect....

Is there anything else (which I have searched but can't find)   that I need to change ?

**Comment by Jyers** [comment, marked as answer]
Sometimes EEPROM can be a little... well... clingy. After flashing go to the control menu and select reset to defaults and then store settings, that should make sure it uses your new values.

**Reply by hellraiserz32** [reply]
Thank you for replying ...I will try that....🙂

**Comment by smooreace** [comment]
Check your nozzle. The setting works fine. But that's an INSANE e-step.
What extruder are you using?

**Reply by hellraiserz32** [reply]
Thank you for replying  .....Please read first post. The estep is in the guide of the HEMERA extruder instructions. If I swap back to the stock firmware with the 409 mod it works. It's something in the Jyers that's not allowing the change of E step...

**Reply by hellraiserz32** [reply]
Here is the guide .. check out item 24 .

https://e3d-online.zendesk.com/hc/en-us/articles/360015891198-Hemera-Creality-Ender-3-V2-Firmware

**Reply by smooreace** [reply]
Sorry, I wasnt familliar with "Hemera" being a name or relevance. Didnt know it was actually a DD model.

But.. I have my e-steps set via config.h, and have fine-tuned it more via on-screen menu as well. Both working great. You can also set it via gcode. But you would have to look up the actual command.
